Sources/OutlineView/TreeMap.swift

```diff
@@ -0,0 +1,223 @@
+
+/// An object representing the state of an OutlineView in which the content
+/// of the OutlineView is Identifiable. The TreeMap should be able to stay
+/// in sync with the OutlineView by reacting to data updates in the OutlineView.
+class TreeMap<D: Hashable> {
+        
+    struct Node {
+        var parentId: D?
+        var isLeaf: Bool
+        
+        // childIds can be nil if `isLeaf` == true, or if the item is not expanded.
+        // If childIds is not nil, that means the item is expanded.
+        var childIds: [D]?
```

Comment:
Acronyms are uniformly capitalised based on the first letter. `childIds` -> `childIDs`.

From swift.org:
>**Follow case conventions.** [...]
[Acronyms and initialisms](https://en.wikipedia.org/wiki/Acronym) that commonly appear as all upper case in American English should be uniformly up- or down-cased according to case conventions:
```swift
var utf8Bytes: [UTF8.CodeUnit]
var isRepresentableAsASCII = true
var userSMTPServer: SecureSMTPServer
```

https://www.swift.org/documentation/api-design-guidelines/#general-conventions

Reply:
Thanks for pointing that out. Fixed here: https://github.com/Sameesunkaria/OutlineView/pull/10/commits/7f8c119c65a508bbb5dd2bbed50265f47715628d